packages/compass-e2e-tests/smoke-test.ts

```diff
@@ -25,44 +32,47 @@ const argv = yargs(hideBin(process.argv))
     type: 'string',
     default: process.env.EVERGREEN_BUCKET_KEY_PREFIX,
   })
-  .option('devVersion', {
+  .option('version', {
     type: 'string',
     // For dev versions we need this from evergreen. For beta or stable (or by
     // default, ie. when testing a locally packaged app) we get it from the
     // package.json
     default: process.env.DEV_VERSION_IDENTIFIER,
+    description:
+      'Will be read from packages/compass/package.json if not specified',
   })
-  .option('isWindows', {
-    type: 'boolean',
-    default: process.env.IS_WINDOWS === 'true',
-  })
-  .option('isOSX', {
-    type: 'boolean',
-    default: process.env.IS_OSX === 'true',
-  })
-  .option('isRHEL', {
-    type: 'boolean',
-    default: process.env.IS_RHEL === 'true',
-  })
-  .option('isUbuntu', {
-    type: 'boolean',
-    default: process.env.IS_UBUNTU === 'true',
+  .option('platform', {
+    type: 'string',
+    choices: ['win32', 'darwin', 'linux'],
+    demandOption: true,
+    default: process.env.PLATFORM ?? process.platform,
   })
   .option('arch', {
     type: 'string',
     choices: ['x64', 'arm64'],
     demandOption: true,
     default: process.env.ARCH ?? process.arch,
   })
+  .option('package', {
+    type: 'string',
+    choices: [
+      'windows_setup',
```

Comment:
These are based on the naming convention in hadron-build. ie. in theory you can just do `${context.package}_filename` to look up the filename for each of them.